game.py

```diff
@@ -102,6 +103,93 @@ def _action_to_steps_move(self, action: Action) -> list[int]:
     def _steps_move_to_action(self, steps_move: list[int]) -> Action:
         """Convert a pydraughts move to something usable by the model"""
         return (steps_move[0] - 1) * self.BOARD_SIZE + steps_move[1] - 1
+    
+    def get_count(self, Player) -> int:
```

Comment:
I think these functions related to the heuristics player should be moved to their own .py file (heuristics.py), instead of being added to the game.py class